src/pages/guides/concepts/migration.md

```diff
@@ -0,0 +1,420 @@
+title: Migration Guide
+description: This is a migration guide for V3 to V4.
+contributors:
+ - https://github.com/nimithajalal
+---
+
+# Migration Guide: Adobe Express Embed SDK V3 to V4
+ 
+This guide will assist you in updating your Adobe Express Embed SDK integration from V3 to v4.
+
+## Overview
+
+Adobe Express Embed SDK V4 introduces a more verbose set of APIs, simplifies parameters, and removes redundancies.
+
+SDK V4 was developed to streamline the existing APIs and create a scalable model to accommodate the expanding range of SDK Target Applications. Accordingly, APIs are categorized into three distinct workflows, each corresponding to one of the SDK's supported target applications. This categorization further refines the APIs, making them more detailed and aligned with user intent. The transition from V3 to V4 brought several changes.
+
+## Workflow API changes
+
+In V3, there were three main APIs: `createDesign`, `EditDesign`, and `openQuickActions`.
+
+These APIs differentiated user intent by analyzing parameters provided by partner teams, subsequently initiating the appropriate workflow. However, this approach increased the parameters for each API, as they were designed to cater to multiple user intents. 
+
+V4 addressed this issue by segmenting the APIs into workflows based on user intent and providing more descriptive API names.
+
+### V4 Key changes
+
+#### Introducing `Module` APIs through Module Workflow
+
+The introduction of Module APIs such as `editImage` and `createImageFromText`.
+
+#### Editor workflow
+
+- `create` initiates a workflow using a blank canvas.  
+- `createWithAsset` allows an image asset to be preloaded onto the canvas.  
+- `createWithTemplate` enables starting with a given Adobe Express template ID.  
+- `edit` allows modification using an existing Adobe Express Document ID.  
+
+#### Quick action workflow
+
+`openQuickAction` has been decommissioned and replaced with individual `quick action-named` APIs such as `cropImage`, eliminating the need for partners to submit a quick action ID.
+
+## Workflow API Changes
+
+For all the workflow APIs, we will now have four parameters.
+
+The four parameters are:
+
+- `DocConfig` - Describes the starting point of a workflow, for example, canvas size for Express Editor.
+- `AppConfig` - properties that configure the target application starting behaviour. 
+- `ExportConfig` - properties that govern the export behaviour of a workflow.
+- `ContainerConfig` - UI properties that customize the SDK container.
+
+UserInfo, AuthInfo, and Callbacks have been removed from a workflow API level and added at the `initialize` level.
+
+The following diagram shows how the previous API relates to the current new APIs:
+
+![V3 V4 comparison](../concepts/v3-v4.png)
+
+### Workflow API example
+
+**V3**: `Editor create API: ccEverywhere.createDesign(inputParams);`
+
+**V4**: `Editor create API: ccEverywhere.editor.create(docConfig, appConfig, exportConfig, containerConfig);`
```

Comment:
To avoid repetition of "Editor create API" and move it out of the monospace, an alternative could be:

### Editor Workflow API example

**V3**: `ccEverywhere.createDesign(inputParams);`

**V4**: `ccEverywhere.editor.create(docConfig, appConfig, exportConfig, containerConfig);`